### PR TITLE
Add receipt sending endpoint

### LINKED ISSUE #4988: Feature to send receipt to attendee via email
**Is your feature request related to a problem? Please describe.**
After an order is created there should be a feature to send the receipt of the order to the attendee. In the Eventbrite Organizers app, there is a feature to send a receipt of the order to the email of the attendee through a common email ```orders@eventbrite.com```.

**Screenshot**
ezgif com-resize 27

**Willing to work on this issue**
No. Please anyone take this up.

### PR DESCRIPTION
**Pull request by schedutron**
Fixes #4988

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [ ] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This PR adds an endpoint to send receipts to attendees corresponding to a given order.

#### Changes proposed in this pull request:
- Implementation of a new endpoint

#### Screenshots
screen shot 2018-07-15 at 12 45 22 am
screen shot 2018-07-15 at 12 45 01 am

I'm not sure whether it's more appropriate to send order identifier instead of order id. @srv-twry @mayank8318 @pradeepgangwar @sridharjajoo Please let me know regarding that choice. I'll add the related docs and tests after that :)

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -114,24 +114,26 @@ def create_app():
...
         app.register_blueprint(api_v1)
+        app.register_blueprint(celery_routes)
         app.register_blueprint(event_copy)
-        app.register_blueprint(upload_routes)
         app.register_blueprint(export_routes)
         app.register_blueprint(import_routes)
-        app.register_blueprint(celery_routes)
-        app.register_blueprint(auth_routes)
-        app.register_blueprint(event_statistics)
+        app.register_blueprint(upload_routes)
```
[the remark this answers is not in the document]
Comment (schedutron) [reply app/__init__.py]: I thought you'd ask to sort the imports after I added the blueprint registration for the added route. 😅

[the remark this answers is not in the document]
Comment (srv-twry) [reply app/__init__.py]: Yeah that is okay but I am talking about sorting the `app.register_blueprint..` statements. Please don't make these kind of changes. Imports is fine but not good in code.

[the remark this answers is not in the document]
Comment (schedutron) [reply app/__init__.py]: So should I reset the order for `app.register_blueprint...` statements?

[the remark this answers is not in the document]
Comment (srv-twry) [reply app/__init__.py]: Yes.

**Comment by schedutron** [comment]
@srv-twry I added `ConflictException` but the endpoint is now throwing an HTTP 500 due to it. The response is not the error message but a traceback.

**Comment by srv-twry** [comment]
@schedutron Okay, revert it back to 403. We don't want HTTP 500.

**Comment by srv-twry** [comment]
@schedutron Just noticed this. I wanted you to change the error code 409. Not wrap it inside ConflictException. Changing to 409 should work.

**Comment by schedutron** [comment]
@srv-twry Is order id fine or should order identifier be used? Also, should I submit tests and docs in another PR?

**Comment by srv-twry** [comment]
We tend to use `identifier` for order. so better to use that. I think you can add docs and subsequently the dredd tests in this PR only.

**Comment by sridharjajoo** [comment]
@schedutron Can you update the docs? Thanks.

**Pull request merged**